project/profiles/capacity4more/modules/c4m/content/c4m_content_project/c4m_content_project.module

```diff
@@ -401,6 +401,24 @@ function c4m_content_project_c4m_offcanvas_trigger_label_left_alter(&$trigger_la
  * Implements hook_c4m_cta_block().
  */
 function c4m_content_project_c4m_cta_block() {
+  global $user;
+
+  // Only add items if there is an OG group.
+  if (!$group = c4m_og_current_group()) {
+    return;
+  }
+
+  // Prevent from 'group' nodes to run this logic.
+  if ($group->type != 'project') {
+    return;
+  }
+
+  // A blocked (banned) user should not be able to see the CTA block.
+  $user_role = c4m_user_og_get_role_in_group($user, $group);
```

Comment:
Replicated logic from group dashboard.
Thought that if it was required for group, should be required for project as well.
It's logical to block follow options for 'blocked' users.